tasks/interactive-bots.md

```diff
@@ -157,7 +166,10 @@ messages to users.
   The bot now responds to `@link` and `@resource`.
 
 
-* Edit the `handle_message`:
+* Create a `links` stream.
```

Comment:
I'm not sure if the change is required - I mean it probably was intentional to keep the followup stream inside the description